### PR TITLE
Removed some dead code, separated CAM deployment structure population…

### PR DESCRIPTION
**Pull request by bill-dall**
… to a separate function, single-sourced more strings.

### CODE REVIEW

**Comment by bill-dall** [comment]
When I was flipping through the file yesterday I decided to so some cleanups for maintainability.

File: Deploy-CAM.ps1 [hunk 1]
```diff
@@ -1095,6 +967,77 @@ function New-CAMAppSP()
...
 }
 
+#creates cam deployment info structures and populates keyvault
+function New-CamDeploymentInfo()
+{
+	param(
+	    [parameter(Mandatory=$true)] 
+	    $subscriptionId, # Azure subscription being operated on
+
+		[parameter(Mandatory=$true)]
```
Comment (ivpetrov87) [inline Deploy-CAM.ps1]: is this a mixed space - tab indent issue?

File: Deploy-CAM.ps1 [hunk 2]
```diff
@@ -1186,6 +1129,11 @@ function Deploy-CAM()
 	$CAMConfig.internal.rwLocalSecretName = 'remoteWorkstationLocalAdminPassword'
 	$CAMConfig.internal.csLocalSecretName = 'connectionServiceLocalAdminPassword'
 	$CAMConfig.internal.saSasTokenSecretName = "userStorageAccountSaasToken"
+	$CAMConfig.internal.blobUriSecretName = "userStorageAccountUri"
```
Comment (ivpetrov87) [inline Deploy-CAM.ps1]: If this is how we are saving them into key vault, i think we should let primesoft know that the instructions for retrieving this information for the front end should probably change, the examples i gave were using these as separate secrets

File: Deploy-CAM.ps1 (hunk 1 above)
Comment (bill-dall) [reply Deploy-CAM.ps1]: Yeah - I've got some more edits coming in a subsequent PR. I'll look at unifying on I guess tabs in this case based on the editor defaults, for that PR.

**Pull request merged**